stacks_queues/stack.py

```diff
@@ -21,18 +21,30 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+        if self.store == None:
```

Comment:
`self.store` is initialized to a LinkedList instance so it shouldn't technically be None. I recommend using `empty` instead!
```suggestion
        if self.store.empty():
```